tensornetwork/backends/base_backend.py

```diff
@@ -135,7 +135,12 @@ def rq_decomposition(
     raise NotImplementedError(
         "Backend '{}' has not implemented rq_decomposition.".format(self.name))
 
-  def concat(self, values: Sequence[Tensor], axis) -> Tensor:
+  def shape_concat(self, values: Sequence[Tensor], axis) -> Tensor:
+    """Concatenate a sequence of tensors together about the given axis."""
```

Comment:
Can you change the description here to be explicitly only for shape calculations?

Reply:
Added to the description, now read: 
```
"""Concatenate a sequence of tensors together about the given axis,
intended only for use in shape calculations"""
```